extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -104,7 +104,8 @@ function render_player( $player_data, $attributes ) {
 
 	// Genereate a unique id for the block instance.
 	$instance_id = wp_unique_id( 'jetpack-podcast-player-block-' );
```

Comment:
This will most generate a PHPCS warning since the equal sign is not aligned with the one below.

Reply:
I aligned the equal sign with the one below it 👍